File: ArtFlow/ArtFlow.Core/Entities/Exhibition.cs
using ArtFlow.Core.Abstractions;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ArtFlow.Core.Entities
{
    public class Exhibition : IBaseEntity
    {
        public int ExhibitionId { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string OrganiserId { get; set; }

        public User Organiser { get; set; }

        public DateTimeOffset HostedOn { get; set; }

        public string Adress { get; set; }

        public List<ExhibitionArtpiece> ExhibitionArtpieces { get; set; } = new List<ExhibitionArtpiece>();

        public List<Order> Orders { get; set; } = new List<Order>();

        public List<Room> Rooms { get; set; } = new List<Room>();
    }
}

File: client-app/src/app/models/exhibition.ts
import { store } from "../stores/store";
import { User } from "./user";

export interface Exhibition {
    exhibitionId: number;
    name: string;
    description: string;
    organiserId: string;
    organiser: User;
    startDate: Date;
    endDate: Date;
    adress: string;
}

export class Exhibition implements Exhibition {
    constructor() {
        this.name = "";
        this.description = "";
        this.organiserId = store.userStore.currentUser!.id;
        this.organiser = store.userStore.currentUser!;
        this.startDate = new Date();
        this.endDate = new Date();
        this.adress = "";
    }
}